### ISSUE TITLE
Video loading error on Imgur

### ISSUE THREAD

**Issue by abranson** [accumulator/Quickddit#42]
All videos hosted on Imgur are failing to load, showing "Error loading video" and "Problem finding stream URL". Console says:

```
[W] QPython::emitError:487 - Unhandled PyOtherSide error: Return value of PyObject call is NULL: Traceback (most recent call last):

  File "/usr/share/harbour-quickddit/youtube_dl/YoutubeDL.py", line 776, in extract_info
    ie_result = ie.extract(url)

  File "/usr/share/harbour-quickddit/youtube_dl/extractor/common.py", line 434, in extract
    ie_result = self._real_extract(url)

  File "/usr/share/harbour-quickddit/youtube_dl/extractor/imgur.py", line 67, in _real_extract
    expected=True)

youtube_dl.utils.ExtractorError: No sources found for video 8ONYfDM. Maybe an image?
```

I think this might be fixed here: https://github.com/rg3/youtube-dl/pull/16537

Edit: Yeah, that fixes it :)

**Comment by PureTryOut** [comment]
How exactly would I apply the youtube-dl fix for myself?

**Comment by abranson** [comment]
You can edit the file yourself: /usr/share/harbour-quickddit/youtube_dl/extractor/imgur.py

Paste this over the top: https://github.com/jwilk-forks/youtube-dl/blob/735646fc67cf17f23bb2822a74156a88a5ee5263/youtube_dl/extractor/imgur.py

**Comment by accumulator** [comment]
fixed in v1.8.1
